app/controllers/api/v1x0/portfolio_items_controller.rb

```diff
@@ -2,6 +2,8 @@ module Api
   module V1x0
     class PortfolioItemsController < ApplicationController
       include Api::V1x0::Mixins::IndexMixin
+      include Api::V1x0::Mixins::RBACMixin
```

Comment:
Is there value in moving this mixin into the `ApplicationController`? The before action makes sense in each controller obviously, but since we're adding more and more RBAC control around, it seems nice to DRY up this `include Api::V1x0::Mixins::RBACMixin'

Reply:
@syncrou Done